src/content/docs/develop/Plugins/develop-mobile.mdx

```diff
@@ -260,6 +260,20 @@ impl<R: Runtime> <plugin-name;pascal-case><R> {
 }
 ```
 
+:::note
```

Comment:
should this block perhaps be in the android block above so it's next to the other kotlin code?

Reply:
good point, move! 👌